Tuist/Dependencies/Lockfiles/Package.resolved

```diff
@@ -1,250 +1,247 @@
 {
-  "object": {
```

Comment:
This uses the new format of Xcode 13.3, but I think we have not bumped the CI yet (we should do it as soon as we find some time)

Reply:
Right @danyf90! Let me try to see what happens if I change to Xcode 13.3 here!